src/index.ts

```diff
@@ -2,12 +2,14 @@ import { Stream, Sink, ScheduledTask, Time, Scheduler, Disposable } from '@most/
 import { MulticastSource } from '@most/core'
 import { asap, cancelTask } from '@most/scheduler'
 
-export const hold = <A>(stream: Stream<A>): Stream<A> =>
+export interface MulticastStream<A> extends Stream<A> {}
```

Comment:
I'd probably either put this into `@most/core` for the return type of `multicast` or rename this to something `HoldStream` or `HeldStream`.

Reply:
Yes, it could belong in `@most/core`. I like the term `Multicast...` because it is well known from other observable/stream libraries, and decently describes the nature of the stream.